developer-tools.md

```diff
@@ -435,6 +435,46 @@ Error:(147, 9) value q is not a member of StringContext
         q"""
         ^ 
 ```
+<h4>Remote Debug Unit Test</h4>
```

Comment:
I think we should make it more general: "Debug Spark Remotely".

Running SBT unit is just one way to trigger the IntelliJ debugging, users can go other ways like running pyspark tests, as long as they set the java options correctly.

Reply:
+1 for @cloud-fan 's point.